addons/website_sale/static/tests/tours/mega_menu_eCommerce.js

```diff
@@ -0,0 +1,96 @@
+import {
+    clickOnExtraMenuItem,
+    registerWebsitePreviewTour,
+} from '@website/js/tours/tour_utils';
+
+const toggleMegaMenu = (stepOptions) =>
+    Object.assign(
+        {},
+        {
+            content: "Toggles the mega menu.",
+            trigger: ":iframe .top_menu .nav-item a.o_mega_menu_toggle",
+            run(helpers) {
+                // If the mega menu is displayed inside the extra menu items, it should
+                // already be displayed.
+                if (!this.anchor.closest(".o_extra_menu_items")) {
+                    helpers.click();
+                }
+            },
+        },
+        stepOptions
+    );
+
+registerWebsitePreviewTour(
+    "mega_menu_eCommerce",
+    {
+        url: "/",
+        edition: true,
+    },
+    () => [
+        // Add a megamenu item to the top menu.
+        {
+            content: "Click on a menu item",
+            trigger: ":iframe .top_menu .nav-item .nav-link [data-oe-model='website.menu']",
+            run: "click",
+        },
+        {
+            content: "Click on 'Link' to open Link Dialog",
+            trigger: ".o-we-linkpopover .js_edit_menu",
+            run: "click",
+        },
+        {
+            trigger: ".o_website_dialog:visible",
+        },
+        {
+            content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+            trigger: ".modal-body a:eq(1)",
+            run: "click",
+        },
+        {
+            content: "Write a label for the new menu item",
+            trigger: ".modal-dialog .o_website_dialog input",
+            run: "edit Megaaaaa!",
+        },
+        {
+            content: "Confirm the mega menu label",
+            trigger: ".modal .modal-footer button:contains(Continue)",
+            run: "click",
+        },
+        {
+            trigger:
+                '.oe_menu_editor [data-is-mega-menu="true"] .js_menu_label:contains("Megaaaaa!")',
+        },
+        {
+            content: "Save the website menu with a new mega menu",
+            trigger: ".modal .modal-footer button:contains(save)",
+            run: "click",
+        },
+        {
+            trigger: "body:not(:has(.modal))",
+        },
+        {
+            trigger: ".o_builder_sidebar_open",
+        },
+        {
+            trigger: ".o_builder_open .o_website_preview:not(.o_is_blocked)",
+        },
+        // Edit a menu item
+        clickOnExtraMenuItem({}, true),
+        toggleMegaMenu({}),
```

Comment:
```suggestion
        {
            content: "Toggles the mega menu.",
            trigger: ':iframe .top_menu .nav-item a.o_mega_menu_toggle',
            run(helpers) {
                // If the mega menu is displayed inside the extra menu items, it should
                // already be displayed.
                if (!this.anchor.closest('.o_extra_menu_items')) {
                    helpers.click();
                }
            },
        },
```
Not really sure why you'd define a `toggleMegaMenu` function if you can just insert the step as is?

Also, in the sales scope, single quotes are still preferred for technical strings in JS.